File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Power.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Power {

    @SerializedName("id")
    @Expose
    public Integer id;
    @SerializedName("essence")
    @Expose
    public Essence essence;
    @SerializedName("tierId")
    @Expose
    public Integer tierId;
    @SerializedName("majorPowerSpell")
    @Expose
    public Spell majorPowerSpell;
    @SerializedName("minorPowerSpell")
    @Expose
    public Spell minorPowerSpell;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Power.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Power(
    @SerializedName("id")
    @Expose
    var id: Int? = null,

    @SerializedName("essence")
    @Expose
    var essence: Essence? = null,

    @SerializedName("tierId")
    @Expose
    var tierId: Int? = null,

    @SerializedName("majorPowerSpell")
    @Expose
    var majorPowerSpell: Spell? = null,

    @SerializedName("minorPowerSpell")
    @Expose
    var minorPowerSpell: Spell? = null
)